server/src/main/java/org/opensearch/cluster/service/ClusterApplierService.java

```diff
@@ -125,6 +126,10 @@ public class ClusterApplierService extends AbstractLifecycleComponent implements
     private NodeConnectionsService nodeConnectionsService;
     private final ClusterManagerMetrics clusterManagerMetrics;
 
+    public ClusterApplierService(String nodeName, Settings settings, ClusterSettings clusterSettings, ThreadPool threadPool) {
```

Comment:
Can we have a testcase that uses the no-op registery using this constructor and emits metrics ?

Reply:
Updated in the revision.